lib/internal/util.js

```diff
@@ -614,6 +616,32 @@ function exposeGetterAndSetter(target, name, getter, setter = undefined) {
   });
 }
 
+let undici;
+/**
+ * Lazy load Undici module by proxy every class with serializable
+ * so that its instances can be recognized as platform objects
+ */
+function lazyUndici() {
+  if (!undici) {
+    undici = require('internal/deps/undici/undici');
+    for (const mod of [
+      'WebSocket', 'EventSource', 'FormData', 'Headers',
+      'Request', 'Response', 'MessageEvent', 'CloseEvent']) {
+
+      undici[mod] = new Proxy(undici[mod], {
+        __proto__: null,
+        construct(target, args, newTarget) {
+          const obj = ReflectConstruct(target, args, newTarget);
+          // 0 means not cloneable, nor transferable
+          obj[transfer_mode_private_symbol] = 0;
+          return obj;
+        },
+      });
+    }
+  }
+  return undici;
+}
+
```

Comment:
`internal/util` provides as a `getLazy`, IMO that should be used instead.

Reply:
We don't want to double proxy `undici` when it's already loaded. Using `??=` or `getLazy` will not be able to achieve that.

Reply:
No, I meant within this function
```suggestion
/**
 * Lazy load Undici module by proxy every class with serializable
 * so that its instances can be recognized as platform objects
 */
function lazyUndici() {
  return getLazy(() => { ... });
}
```

Reply:
Ah, I get it now. That's a good suggestion.

Reply:
Looks like the function closure from `getLazy` caused some tests failed. I couldn't get around with it, so I reverted that change.